.dev/scripts/generate_changelog.py

```diff
@@ -184,6 +183,12 @@ def generate_module_section(key, prs):
     cache_grp = parser.add_mutually_exclusive_group()
     cache_grp.add_argument("--save", type=Path, help="Save PR data fetched from GitHub")
     cache_grp.add_argument("--load", type=Path, help="Load PR data from a file")
+    parser.add_argument(
+        "--with-excluded",
```

Comment:
`--with-misc` or `--keep-unimportant`, etc. seem more self-explanatory

Reply:
This was an allusion to now-removed "changelog: exclude" label. But you are right, `--with-misc` is better.